mu4/scenes/inspector/iinspectoradapter.h

```diff
@@ -0,0 +1,76 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2020 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+
+#ifndef IINSPECTORADAPTER_H
+#define IINSPECTORADAPTER_H
+
+#include <QVariant>
+
+#include "modularity/imoduleexport.h"
+
+#include "libmscore/style.h"
+
+namespace Ms {
+class Score;
+}
+
+namespace mu {
+namespace scene {
+namespace inspector {
+class IInspectorAdapter : MODULE_EXPORT_INTERFACE
```

Comment:
I really hope that we could do without this because the Inspector was recently designed and I thought it would be easy to use. (for example, unlike Palette, where the code is more confusing)